lib/devise/models/http_header_authenticatable.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'devise/strategies/http_header_authenticatable'
+
+module Devise
+  module Models
+    module HttpHeaderAuthenticatable
+      extend ActiveSupport::Concern
+
+      def after_database_authentication; end
```

Comment:
I'm assuming we're noop-ing this so that Devise doesn't do its usual thing. Could you add a comment in here to confirm that?